packages/client/src/typescript.ts

```diff
@@ -109,6 +110,14 @@ export interface ITsConstructor extends ITsDocBase, ITsCallable {
     kind: Kind.Constructor;
 }
 
+export interface ITsAccessor extends ITsDocBase {
+    kind: Kind.Accessor;
+    /** Value type for the set accessor, or null if set accessor is not defined. */
+    valueType: string | null;
```

Comment:
get and set accessors are required to have the same type, so just one `type: string` is necessary here. and then we don't need to handle the undefined case.